src/recipe/recipe.controller.ts

```diff
@@ -1,8 +1,20 @@
-import { Controller } from '@nestjs/common';
+import { Controller, Get, NotFoundException, Param, UseGuards } from '@nestjs/common';
+import { BearerAuthGuard } from '../auth/strategies/bearer.strategy';
+import { RecipeDto } from './dto/recipe.dto';
 
 import { RecipeService } from './recipe.service';
 
 @Controller('recipe')
 export class RecipeController {
   constructor(private recipeService: RecipeService) {}
+
+  @UseGuards(BearerAuthGuard)
+  @Get(':_id')
```

Comment:
I think `:id` is a better param name than `:_id`

Reply:
I'm not sure if you can change it without breaking the contract/responses
I was thinking about using `@Expose({ name: 'id' })` on `RecipeDto`, but I'm not sure how it would affect other things.

Reply:
This would be a change for a separate ticket